### PR TITLE
fix issue #10764

### PR DESCRIPTION
**Pull request by pkirill99**
fix issue https://github.com/yiisoft/yii2/issues/10764

### CODE REVIEW

File: framework/widgets/BaseListView.php [hunk 1]
```diff
@@ -133,7 +133,11 @@ public function run()
 
         $options = $this->options;
         $tag = ArrayHelper::remove($options, 'tag', 'div');
-        echo Html::tag($tag, $content, $options);
+        if($tag !== false) {
```
Comment (SilverFire) [inline framework/widgets/BaseListView.php]: Might be simplified to one-line with a ternary operator.

**Comment by SilverFire** [comment]
Changelog and tests are required

**Comment by SilverFire** [comment]
I'm closing this PR in favor of changing `Html::tag` behavior as described in https://github.com/yiisoft/yii2/issues/10764#issuecomment-180781200

**Comment by pkirill99** [comment]
but also in BaseListView:

` echo Html::tag($tag, $content, $options);`

maybe

` return Html::tag($tag, $content, $options);`

?

**Comment by cebe** [comment]
there is no real difference https://github.com/yiisoft/yii2/blob/master/framework/base/Widget.php#L107

**Pull request closed without merging**